### DISCUSSION TITLE
Dodona 6.6 released

### DISCUSSION THREAD

**Discussion by jorg-vr** [dodona-edu/feedback-and-support#82] [Announcements]
This release includes the next phase of our homepage redesign: a brand new design for the course cards. With the new design we have tried to display more relevant information and provide a link to the most useful actions for that course.

For students, the progress bar in the header shows how many exercises they have completed in the course. In the card itself, we show the last series they have worked on and one or more exercises.

For teachers, the header contains some key figures about the course, such as the number of series, exercises, submissions and students. In the card itself, we give priority to showing any unanswered questions or pending course registrations. We also show one or more series that students have recently worked on. A layered progress bar shows the progress of the class for each series.

The resulting cards for students:
![image](https://user-images.githubusercontent.com/21177904/215768655-9d20afef-e475-4b8d-80a3-235aff17f889.png)

The resulting cards for teachers:
![image](https://user-images.githubusercontent.com/21177904/215768737-7518372f-1f44-456e-8475-45fa74cad579.png)

---

### 🚀 New Features
* Homepage course cards redesign by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4261
* Show more relevant info for teachers on course cards by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4299
### ✨ Changes
* Allow submitting an annotation with shortcut from title edit field by @chvp in https://github.com/dodona-edu/dodona/pull/4358
### 🐛 Bug Fixes
* Clear scores of empty feedback on submission select by @chvp in https://github.com/dodona-edu/dodona/pull/4333
* Fix missing translation of search action titles by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4337
* Only list unique elements in dropdowns on the repository page by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4338
* Fix error messages when uploading incorrect user labels by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4342
* Don't process mathjax math in the code editor by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4352
* Always display score items in a fixed order by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4354
* Hide output if it is much longer than expected to avoid browser crashes by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4353
* Fix translation missing on submission export by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4357
* Add unique index to series_id for evaluations by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4359
### 🧰 Maintenance
* export.js: remove jquery and convert to typescript by @freyavs in https://github.com/dodona-edu/dodona/pull/4267
* Fix test badge link by @chvp in https://github.com/dodona-edu/dodona/pull/4339
* Add "cite this repo" to GitHub repo by @bmesuere in https://github.com/dodona-edu/dodona/pull/4341
* Rename application_old and only include where relevant by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4374
* Limit the number of notifications requests by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4373

**Full Changelog**: https://github.com/dodona-edu/dodona/compare/6.5.0...6.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dodona-edu/dodona/releases/tag/6.6.0'>6.6.0</a>.</em>

**Comment by bmesuere** [comment]
## What's changed in 6.6.1
### ✨ Changes
* Perform less repository requests on the course page by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4377
* Keep programming languages in memory by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4376
* Cache submission `annotated?` as boolean by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4379
* Only show own repositories on the index page for teachers by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4380
* Allow editing repositories for repository admins by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4381
* Fix spelling and grammar by @bmesuere in https://github.com/dodona-edu/dodona/pull/4386
* Fix multiple repository related paper cuts by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4383
* Valentine status overlay by @tibvdm in https://github.com/dodona-edu/dodona/pull/4400
### 🐛 Bug Fixes
* Add callout success by @bmesuere in https://github.com/dodona-edu/dodona/pull/4385
* Limit the amount of users exposed by the repository admin screen by @jorg-vr in https://github.com/dodona-edu/dodona/pull/4382
### 🧰 Maintenance
* series.js: remove jquery and convert to typescript by @freyavs in https://github.com/dodona-edu/dodona/pull/4266
* Add deploy by label for naos by @bmesuere in https://github.com/dodona-edu/dodona/pull/4384


**Full Changelog**: https://github.com/dodona-edu/dodona/compare/6.6.0...6.6.1
